src/odm_sharing/tools/share.py

```diff
@@ -157,13 +156,14 @@ def get_excel_writer(in_name, debug: bool, org: str, outdir: str,
 
 
 def infer_outfmt(path: str) -> OutFmt:
+    '''returns AUTO when not recognized'''
     (_, ext) = os.path.splitext(path)
     if ext == '.csv':
         return OutFmt.CSV
     elif ext == '.xlsx':
         return OutFmt.EXCEL
     else:
-        quit(f'unable to infer output format from input file ext "{ext}"')
+        return OutFmt.AUTO
```

Comment:
I find it more understandable to return `None`

Reply:
done